scripts/Diffraction/single_crystal/base_sx.py

```diff
@@ -614,6 +618,180 @@ def remove_non_integrated_peaks(peaks):
             EnableLogging=False,
         )
 
+    @staticmethod
+    def plot_integrated_peaks_MD(
+        wsMD: IMDEventWorkspace,
+        peaks: IPeaksWorkspace,
+        filename: str,
+        nbins_max: Optional[int] = 21,
+        extent: Optional[float] = 1.5,
+        log_norm: Optional[bool] = True,
+    ):
+        """
```

Comment:
The purpose of this function is probably blindingly obvious, but is it worth adding a short description to this doxygen string anyway?